docs/developer/communications_guideline.md

```diff
@@ -0,0 +1,42 @@
+# Communications Guidelines 
+
+pyDARN is a collaborative community of engineers, scientists, and programmers that is working towards an easy maintainable python library for data visualizations for SuperDARN. 
+To achieve the collaborative community, constructive feedback will be given to allow the growth of pyDARN. However, toxic and negative communication will not be tolerated, this includes users as well. 
+
+To ensure a constructive positive environment is met please follow the following guidelines for the following situations: 
+
+## General communication 
+
+1. No negative or discriminate comments 
+2. Avoid over talking/ignoring someone's comments - this is a collaborative group, so everyone's thoughts and opinions matter. 
+3. No mocking - comments that do not belong in the conversation, or making fun of someone's profession or skills will not be tolerated
+4. Stay on topic - discussion can easily change and evolve into other topics, please make sure to stay within scope. If scope creep happens try to relocate that discussion to another *Issue* or discussion point
+
+
+## Issues 
+1. Keep comments sweet and to the point - reading lengthy explanations can be overwhelming and encourage scope creep
+2. No scope creeping - If scope creep occurs, make another *Issue* on that topic
+3. Take individual discussion offline - If the issue involves a lengthy discussion with a group or two individuals then do it offline from the repository to avoid annoying others with multiple GitHub notifications. If a resolution is made from the discussion then summarize it on the *Issue* so others are aware. 
+4. 
+
+## Pull Requests 
+1. Its all about the code - keep comments focused on the code and avoid individual comments 
+2. Purpose solutions when pointing out problems - code is personal to the developer so pointing out problems can be overwhelming. To help prevent defensive behaviour try to suggest solutions to their problems. 
```

Comment:
This can be problematic and act as a barrier to new code reviewers.  What should someone do if they see a problem, but are unsure about a solution?

Reply:
Then you explain what you are expecting should be, or if it is a bug, then ask a question to confirm that was not an intended behavior or if something needs to be documented as the fix is not possible or doable at the time. 

The reason I ask this so that there is no miss communication, wasted time, or waiting on a response. As we have already had this happen and can be frustrating for both sides or delay progress. 

In code reviews, if you think you see a bug, then make sure to test it to make sure it is a bug and then show the test case in a comment and link to the lines you think are the problem. Thus the developer can investigate and potentially fix it if they see a problem. Or provide an explanation that we later can close if there is no follow up. 

I will explain this more in the documentation when it gets released. 

Thank you for the question!